src/bp/ui-studio/src/web/components/Layout/StatusBar/TrainingStatus.tsx

```diff
@@ -0,0 +1,99 @@
+import { Button } from '@blueprintjs/core'
+import axios from 'axios'
+import { NLU } from 'botpress/sdk'
+import { lang } from 'botpress/shared'
+import cx from 'classnames'
+import _ from 'lodash'
+import React, { FC, useEffect, useState } from 'react'
+import { connect } from 'react-redux'
+import { RootReducer } from '~/reducers'
+
+import style from './style.scss'
+
+interface Props {
+  trainSession: NLU.TrainingSession
+}
+
+// TODO change this url for core ?
+const BASE_NLU_URL = `${window.BOT_API_PATH}/mod/nlu`
+
+const TrainingStatusComponent: FC<Props> = (props: Props) => {
```

Comment:
seems like this component doesn't fetch the initial "needs training" status

Reply:
fixed